packages/fluentui/react-northstar/src/themes/teams/components/Checkbox/checkboxStyles.ts

```diff
@@ -120,11 +120,13 @@ export const checkboxStyles: ComponentSlotStylesPrepared<CheckboxStylesProps, Ch
     ...(p.checked &&
       p.checked && {
         borderColor: v.checkedBorderColor,
+        backgroundColor: v.checkedBackground,
```

Comment:
should this go up to line 121, before the p.checked && {

Reply:
Why are there 2 `p.checked` statements?

Reply:
Thanks, @notandrew, and @codepretty, have done some code clean up, please take a look.